siteapp/settings.py

```diff
@@ -116,9 +126,10 @@ def make_secret_key():
 	'django.contrib.auth.middleware.AuthenticationMiddleware',
 	'django.contrib.messages.middleware.MessageMiddleware',
 	'django.middleware.clickjacking.XFrameOptionsMiddleware',
+	'pyinstrument.middleware.ProfilerMiddleware',
 ]
 if environment["debug"] and os.path.exists(os.path.join(os.path.dirname(__file__), 'helper_middleware.py')):
-	MIDDLEWARE_CLASSES.append(primary_app+'.helper_middleware.DumpErrorsToConsole')
```

Comment:
Are we sure about this change? Why was this previously `MIDDLEWARE_CLASSES`?

Reply:
so since Django 2.0 MIDDLEWARE_CLASSES is MIDDLEWARE

Reply:
Yeah MIDDLEWARE_CLASSES is Django 1.9 or less MIDDLEWARE starts in Django 2.0